sagemaker-clarify/fairness_and_explainability/fairness_and_explainability_boto3.ipynb

```diff
@@ -0,0 +1,1541 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fairness and Explainability with SageMaker Clarify using AWS SDK for Python (Boto3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Runtime\n",
+    "\n",
+    "This notebook takes approximately 30 minutes to run.\n",
+    "\n",
+    "## Contents\n",
```

Comment:
I found that the latest SageMaker Studio has a side panel displaying the table of content auto-generated from the notebook sections. It seems a new feature of JupyterLab3.

Do you think we can get rid of the in-place table of content?

If you want to keep it, could you double check if it is up-to-date or not?

Reply:
I kept it as it would be useful for notebook instances, I myself sometimes prefer executing on notebook instances instead of studio UI.

Reply:
I have updated the contents.